Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -482,6 +486,20 @@ static HRESULT _createPathReadyForFigure(CGPathRef previousPath,
     return S_OK;
 }
 
+static const CGFloat sc_zeroAngleThreshold = .00001;
+// This function will return a normalized angle in radians between 0 and 2pi. This is to standardize
+// the calculations for arcs since 0, 2pi, 4pi, etc... are all visually the same angle.
+static CGFloat __normalizeAngle(CGFloat originalAngle) {
+    CGFloat returnAngle = fmod(originalAngle, 2 * M_PI);
```

Comment:
I'm not certain what this function is doing anymore.

Does it boil down to

```c++
return std::min(abs(angle), 2 * M_PI);
```

?